beeswax/base/eventid.proto

```diff
@@ -12,6 +12,7 @@ message EventId {
   optional uint32 hostid = 1;    // 32 bits of md5 hash
   optional uint32 tid = 2;       // Thread Id from gettid()
   optional uint64 timestamp = 3;
+  optional uint32 imp_idx = 4; // Impression index within the request
```

Comment:
super nit and I realize you didn't make the change, but why do we have comments at the end of the line here instead of above?

Reply:
I'm ok with removing the comment. At a minimum there is no need to be bound to it.